CI-CD.md

```diff
@@ -0,0 +1,34 @@
+# CI/CD 
+## PR-Validation
+### CodeQL
+
+### Overview
+CodeQL is a semantic code analysis engine that helps identify vulnerabilities and coding errors in the source code. 
+In our project, CodeQL is integrated into the GitHub Actions workflow for pull request validation.
+
+### Location
+The CodeQL configuration is located in the GitHub Actions workflow file:
+`.github/workflows/pr-validation.yml`
+
+### Configuration Details
+- **Languages Analyzed**: Java (Note: Kotlin is also analyzed as Java)
+- **Query Sets**:
+    - `security-extended`
+    - `security-and-quality`
```

Comment:
Waarom deze twee? En waarom niet meer? Elke keuze kan goed zijn, zolang het maar verantwoord is